static/css/impala/forms.less

```diff
@@ -242,6 +242,15 @@ textarea {
     input[type=color] {
         width: 80px;
     }
+    .api-label {
+      font-weight: bold;
+      display: block;
+    }
+    input[type=text] {
+      box-shadow: none;
+      text-align: center;
+      font-size: 15px;
```

Comment:
we use `em` for all font sizes, not pixels, so that users with accessibility needs can adjust the size if needed.